nodebuilder/default_services.go

```diff
@@ -0,0 +1,19 @@
+package nodebuilder
+
+import (
+	"github.com/celestiaorg/celestia-node/das"
+	"github.com/celestiaorg/celestia-node/fraud"
+	"github.com/celestiaorg/celestia-node/nodebuilder/header"
+	"github.com/celestiaorg/celestia-node/share/availability/light"
+	"github.com/celestiaorg/celestia-node/state"
+)
+
+// PackageToDefaultImpl maps a package to its default implementation. Currently only used for method discovery for
+// openrpc spec generation
+var PackageToDefaultImpl = map[string]interface{}{
+	"fraud":  &fraud.ProofService{},
+	"state":  &state.CoreAccessor{},
+	"share":  &light.ShareAvailability{},
+	"header": &header.Service{},
+	"daser":  &das.DASer{},
```

Comment:
Later every one here will rely on module, not the pkg, right?

Reply:
Oh, yes! Since we switch to Internal structs for the auth it will be able to use the API structs actually (it needs a struct where the methods are defined onto)